.github/workflows/test-check.yaml

```diff
@@ -31,7 +31,7 @@ jobs:
       - name: "Clean sparsezoo directory"
         run: rm -r sparsezoo/
       - name: ⚙️ Install dependencies
-        run: pip3 install .[dev] opencv-python
```

Comment:
I think it would be cleaner to separate rudimentary deepsparse tests from server tests, especially now when the codebase for the server will grow quickly. Not a blocking comment, but something to think about.

Reply:
Yeah I could even see the server being in a separate repo